ext/mysqli/mysqli.stub.php

```diff
@@ -256,10 +256,10 @@ public function set_opt(int $option, $value) {}
      */
     public function ssl_set(
         string $key,
-        string $cert,
+        string $certificate,
         string $certificate_authority,
```

Comment:
```suggestion
        string $ca_certificate,
```
As this is what we call it in openssl.